manifests/init.pp

```diff
@@ -2,31 +2,30 @@
 #
 # @example To set up a single nomad server, with several agents attached, on the server.
 #   class { 'nomad':
-#     version     => '1.0.2', # check latest version at https://github.com/hashicorp/nomad/blob/master/CHANGELOG.md
 #     config_hash => {
-#       'region'     => 'us-west',
```

Comment:
Personally I always use quotes in hashes rather than string literals. I'm not sure whether you really need to change this.

Reply:
i normally dont, but reverted it